docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1270 @@
+      hpcc chart repository using Helm, then access the Helm chart values from
+      the charts in that repository.</para>
+
+      <para>For example, when you add the "hpcc" repository, as recommended
+      prior to installing the Helm chart with the following command:</para>
+
+      <programlisting>helm repo add hpcc https://hpcc-systems.github.io/helm-chart
+</programlisting>
+
+      <para>You can now view the HPCC Systems delivered charts and see the
+      values there by issuing:</para>
+
+      <programlisting>helm show values hpcc/hpcc</programlisting>
+
+      <para>You can capture the output of this command, look at how the
+      defaults are configured and use it as a basis for your
+      customization.</para>
+    </sect2>
+
+    <sect2 id="Values-SchemaJSONFile">
+      <title>The values-schema.json</title>
+
+      <para>The <emphasis>values-schema.json</emphasis> is a JSON file that
+      declares what is valid and what is not within the sum total of the
+      merged values that are passed into Helm at install time. It defines what
+      values are allowed, and validates the values file against them. All the
+      core items are declared in the schema file, while the default
+      values.yaml file also contains comments on the most important elements.
+      If you wanted to know what options are available for any particular
+      component then the schema is a good place to start.</para>
+
+      <para>The schema file typically contains (for a property) a name and a
+      description. It will often include details of the type, and the items it
+      can contain if it is a list or dictionary. For instance:</para>
+
+      <programlisting>    "roxie": { 
+      "description": "roxie process",
+      "type": "array"
+      "items": { "$ref": "#/definitions/roxie" }
+    },</programlisting>
+
+      <para>The "items" in this case are a reference ($ref) to another section
+      of the schema. Each plane, in the schema file has a list of properties
+      generally containing a prefix (path), a subpath (subpath), and
+      additional properties. The schema file should be complete, and contain
+      everything required including descriptions which should be relatively
+      self-explanatory.</para>
+
+      <programlisting>  "storagePlane": {
+      "description": "information about an individual storage plane",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "the name of the storage plane",
+          "type": "string"
+        },
+        "prefix": {
+          "description": "either the path for a local mount, or the url prefix",
+          "type": "string"
+        },
+        "subPath": {
+          "description": "optional subdirectory within the mount directory",
+          "type": "string"
+        },
+        "secret": {
+          "description": "optional name of any secret required to access this storage plane",
+          "type": "string"
+
```

Comment:
missing closing brackets from the example.

Reply:
probably better to just choose an example with less properties than the storagePlane to illustrate the the json file, I can always revert, if necessary however using storage seems the entire section there only has the 2 properties and will fit nicely on a single page - think that would suffice?